### PR TITLE
RD-10220: CSV.Parse crashing when columns are of nothing type (empty CSV with a header)

### PR DESCRIPTION
**Pull request by bgaidioz**

### CODE REVIEW

File: snapi-client/src/test/scala/raw/compiler/rql2/tests/regressions/RD10220Test.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  test("""Csv.Parse("stringData\nna\nna", type collection(record(stringData: undefined)), nulls=["na"], skip=1)""")(
+    _ should (typeAs("collection(record(stringData: undefined))") and evaluateTo(
+      "[{stringData: null}, {stringData: null}]"
+    ))
+  )
+
+  test("""Csv.Parse("stringData\n12\nna", type collection(record(stringData: undefined)), nulls=["na"], skip=1)""")(
+    _ should (typeAs("collection(record(stringData: undefined))") and evaluateTo(
+      """[{stringData:Error.Build("failed to parse CSV (null: line 2, col 1), unexpected value found, token '12'")}, {stringData:null}]"""
```
Comment (bgaidioz) [inline snapi-client/src/test/scala/raw/compiler/rql2/tests/regressions/RD10220Test.scala:35]: In JSON there's no such long message, only the unexpected value found.

Comment (bgaidioz) [reply snapi-client/src/test/scala/raw/compiler/rql2/tests/regressions/RD10220Test.scala:35]: To be sorted out separately.

**Review by torcato — APPROVED** [review]
LGTM

**Pull request merged**